Shared/Samples/Create KML multi-track/CreateKMLMultiTrackView.Model.swift

```diff
@@ -0,0 +1,158 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+extension CreateKMLMultiTrackView {
+    /// The view model for the sample.
+    @MainActor
+    final class Model: ObservableObject {
+        /// A map with a streets basemap.
+        let map = Map(basemapStyle: .arcGISStreets)
+        
+        /// The map view's location display.
+        let locationDisplay = LocationDisplay()
+        
+        /// The graphics overlays to display on the map view.
+        var graphicsOverlays: [GraphicsOverlay] {
+            return [trackElementGraphicsOverlay, trackGraphicsOverlay]
```

Comment:
Not a big deal, but we typically omit the `return` keyword for single line property getters.

```suggestion
            [trackElementGraphicsOverlay, trackGraphicsOverlay]
```

Reply:
![Screenshot 2025-03-06 at 4 14 02 PM](https://github.com/user-attachments/assets/de11370f-3cba-412c-9da2-84668c91dc0d)

With `return` removed, I'm seeing this warning. Either suppress the rule or add back `return`.